packages/@dcl/sdk/package.json

```diff
@@ -25,9 +25,9 @@
   "dependencies": {
     "@dcl/amd": "file:../amd",
     "@dcl/build-ecs": "file:../build-ecs",
-    "@dcl/kernel": "1.0.0-2994874542.commit-c3ae489",
-    "@dcl/posix": "^1.0.4",
-    "@dcl/unity-renderer": "^1.0.40531-20220621125654.commit-472137e"
+    "@dcl/kernel": "^1.0.0-3339209749.commit-fa9e5d7",
```

Comment:
@leanmendoza caret?

Reply:
For testing ECS7 it's okay as we're moving fast. Actually we currently are using de query params `&kernel-branch=main&renderer-branch=dev` => https://github.com/decentraland/cli/blob/main/src/commands/start.ts#L204